src/python/tests/core/bot/fuzzers/blackbox/test_data/sample_fuzz_output/fuzz-noncrash-testcase

```diff
@@ -0,0 +1 @@
+2
```

Comment:
Where is this used ?

Reply:
All of these are used by test_fuzz. It's expected to find that real-crash is an interesting crash because of its output, and to ignore fuzz-noncrash-testcase because it has no crash stack in its output.